catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -388,8 +388,8 @@ public void destroy() {
     config.put("sourcePollInterval", sourcePollInterval);
     config.put("scheduleFrequencyList", scheduleFrequencyList);
     config.put("isEditingAllowed", isEditingAllowed);
-    config.put("isCacheDisabled", isCacheDisabled);
-    config.put("disableLocalCatalog", disableLocalCatalog);
+    config.put("isCacheDisabled", !isCacheEnabled);
```

Comment:
Are we not allowed to change the left-hand-side of this to also be consistent?

Reply:
Since this method is used by Intrigue, that would require front-end changes that are out of scope for this ticket.  I agree they should be changed in general, though.

Reply:
Is it not as simple as a find and replace (with some boolean flipping)? If it isn't included here do we have a ticket that handles it?